library/src/actions/jwt/jwt.ts

```diff
@@ -0,0 +1,193 @@
+import type {
+  BaseIssue,
+  BaseValidation,
+  ErrorMessage,
+} from '../../types/index.ts';
+import { _addIssue } from '../../utils/index.ts';
+
+const POSSIBLE_JWT_REGEX = /^(?:[\w-]+\.){2}[\w-]+$/u;
```

Comment:
Usually the regexes are stored inside `/library/src/regex.ts`. But I think the file is used to only store complete public regexes. By complete I mean if the string passes the regex's test, the string is valid. Like I stated (the comment at the top of this file), this regex is just the first part of the whole check, so its better it is a private member than in `/library/src/regex.ts`. Also, by naming it `POSSIBLE_JWT_REGEX` I meant if the string passes the test, there is a possiblity of the string being a valid jwt. If you have a better name for this regex ref, let me know. 

Reply:
This looks good to me!